entrypoint.sh

```diff
@@ -1,18 +1,19 @@
 #!/bin/sh
+set -exo pipefail
 
 KAFKA_NETWORK="gh-kafka-network"
-docker network create $KAFKA_NETWORK --driver bridge
+docker network create ${KAFKA_NETWORK} --driver bridge
 
 echo "Running Zookeeper container"
-docker run -d --name zookeeper --network $KAFKA_NETWORK -p $INPUT_ZOOKEEPER_PORT:2181 -e ALLOW_ANONYMOUS_LOGIN=yes bitnami/zookeeper:$INPUT_ZOOKEEPER_VERSION
+docker run -d --name zookeeper --network ${KAFKA_NETWORK} -p "${INPUT_ZOOKEEPER_PORT}:2181" -e ALLOW_ANONYMOUS_LOGIN=yes "${INPUT_CONTAINER_IMAGE_ZOOKEEPER}:${INPUT_ZOOKEEPER_VERSION}"
 
 echo "Running Kafka container"
-docker run -d --name kafka --network $KAFKA_NETWORK -p $INPUT_KAFKA_PORT:9092 \
+docker run -d --name kafka --network ${KAFKA_NETWORK} -p "${INPUT_KAFKA_PORT}:9092" \
   -e KAFKA_CFG_ZOOKEEPER_CONNECT=zookeeper:2181 \
-  -e KAFKA_CFG_ADVERTISED_LISTENERS="CLIENT://kafka:$INPUT_KAFKA_CLIENT_PORT,EXTERNAL://localhost:$INPUT_KAFKA_PORT" \
+  -e KAFKA_CFG_ADVERTISED_LISTENERS="CLIENT://kafka:${INPUT_KAFKA_CLIENT_PORT},EXTERNAL://localhost:${INPUT_KAFKA_PORT}" \
   -e KAFKA_CFG_LISTENER_SECURITY_PROTOCOL_MAP="CLIENT:PLAINTEXT,EXTERNAL:PLAINTEXT" \
-  -e KAFKA_CFG_LISTENERS="CLIENT://:$INPUT_KAFKA_CLIENT_PORT,EXTERNAL://:$INPUT_KAFKA_PORT" \
+  -e KAFKA_CFG_LISTENERS="CLIENT://:${INPUT_KAFKA_CLIENT_PORT},EXTERNAL://:${INPUT_KAFKA_PORT}" \
   -e KAFKA_CFG_INTER_BROKER_LISTENER_NAME=CLIENT \
   -e KAFKA_CFG_AUTO_CREATE_TOPICS_ENABLE=true \
   -e KAFKA_CFG_OFFSETS_TOPIC_REPLICATION_FACTOR=1 \
-  -e ALLOW_PLAINTEXT_LISTENER=yes bitnami/kafka:$INPUT_KAFKA_VERSION
+  -e ALLOW_PLAINTEXT_LISTENER=yes "${INPUT_CONTAINER_IMAGE_ZOOKEEPER}:${INPUT_KAFKA_VERSION}"
```

Comment:
```suggestion
  -e ALLOW_PLAINTEXT_LISTENER=yes "${INPUT_CONTAINER_IMAGE_kafka}:${INPUT_KAFKA_VERSION}"
```

Reply:
Nice catch, I knew I might have messed up somewhere but I was looking at the inputs not the vars substations.